media-sound/ncmpcpp/ncmpcpp-0.9.2_p20240724.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+NCMPCPP_COMMIT="68daf44032784a5b5b74781a9bf3826053eff4a2"
+
+DESCRIPTION="Featureful ncurses based MPD client inspired by ncmpc"
+HOMEPAGE="
+	https://rybczak.net/ncmpcpp/
+	https://github.com/ncmpcpp/ncmpcpp/
+"
+SRC_URI="https://github.com/ncmpcpp/ncmpcpp/archive/${NCMPCPP_COMMIT}.tar.gz -> ${P}.gh.tar.gz"
+S="${WORKDIR}/${PN}-${NCMPCPP_COMMIT}"
+
+LICENSE="GPL-2"
```

Comment:
can you change this to `GPL-2+`, see https://github.com/ncmpcpp/ncmpcpp/blob/master/src/settings.cpp#L6-L8